tests/test_data/party/pending_surveys.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "batch_no": "be34d7ca-1d06-4f6e-98e3-73b80c911d32",
+        "business_id": "be3483c3-f5c9-4b13-bdd7-244db78ff687",
+        "email_address": "bob@here.com",
+        "is_transfer": true,
+        "shared_by": "f956e8ae-6e0f-4414-b0cf-a07c1aa3e37b",
+        "survey_id": "cb8accda-6118-4d3b-85a3-149e28960c54",
+        "time_shared": "2025-02-05 16:08:58"
+    }
```

Comment:
As mentioned, we could introduce permutations of these to test various scenarios are covered. Fairly easy to do that.